cmake/std/atdm/utils/get_known_system_info.sh

```diff
@@ -100,6 +101,9 @@ elif [[ $realHostname == "mutrino"* ]] ; then
 elif [[ $realHostname == "waterman"* ]] ; then
   hostnameMatch=waterman
   hostnameMatchSystemName=waterman
+elif [[ $realHostname == "vortex"* ]] ; then
+  hostnameMatch=vortex
+  hostnameMatchSystemName=vortex
```

Comment:
Here is the problem.  Change this to:

```
hostnameMatchSystemName=ats2
```

and you should be set.

Then we will add elif blocks for 'lassen', 'rzansel', and 'sierra' and they will also set `hostnameMatchSystemName=ats2`.  Does that make sense?

Reply:
Got it.